docs/general/ad-filtering/create-own-filters.md

```diff
@@ -987,7 +1006,7 @@ Blocking cookies and removing tracking parameters is achieved by using rules wit
 
 :::info Compatibility
 
-- Stealth Mode is available in AdGuard for Windows, Mac, and Android, and AdGuard Browser Extension. All other products will ignore the rules with `$stealth` modifier.
+- Stealth Mode is available in AdGuard for Windows, Mac, and Android, and AdGuard Browser Extension (but not in the MV3 version). All other products will ignore the rules with `$stealth` modifier.
```

Comment:
Please figure out the FINAL wording that we'll be using in the future.

Will we continue to call the Chrome extension "MV3 version"?
Or will we call it Chrome extension instead?

Reply:
Whenever you mention a product, it would be nice to have an anchor link to the section that explains what product it is.